minimint-api/src/db/mod.rs

```diff
@@ -193,12 +193,12 @@ pub enum DecodingError {
     #[error("Key had a wrong length, expected {expected} but got {found}")]
     WrongLength { expected: usize, found: usize },
     #[error("Other decoding error: {0}")]
```

Comment:
Would it make sense to replace `#[error("Other decoding error: {0}")]` with `#[error(transparent)]]` here ?

Reply:
You can't use both.

Reply:
Yes but I'm wondering if "Other decoding error" is necessary because then I'd just replace it